src/app/containers/MediaPlayer/helpers/embedUrl.js

```diff
@@ -1,13 +1,44 @@
+import Url from 'url-parse';
```

Comment:
So this feels like it's adding a lot of logic around url parsing to the embed url logic. This functionality could be useful in other parts of the app.

I don't want to hold up this work as this is needed right now, however could you write an issue to generalise this logic at a later date as I think this would be a good improvement. I am thinking we could pass the override values to the request context further up in the app, that way any URL that needs to change because of that override doesn't have to to reimplement this logic of getting the query strings etc.

Reply:
I have extracted the functions into a new helper file... I think I want to have a new issue/PR for reusing this in `getInitialData`